sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenter.java

```diff
@@ -209,12 +212,47 @@ private void addReleaseIfNotAlreadyInstalled(Release release, Set<Release> insta
   public List<SonarUpdate> findSonarUpdates() {
     List<SonarUpdate> updates = new ArrayList<>();
     SortedSet<Release> releases = sonar.getReleasesGreaterThan(installedSonarVersion, installedSonarProduct);
+    if (installedSonarProduct == Product.SONARQUBE_COMMUNITY_BUILD) {
+      Release releaseToUpdateTo = findPaidReleaseToUpdateTo();
+      if (releaseToUpdateTo != null) {
+        releases.add(releaseToUpdateTo);
+      }
+    }
     for (Release release : releases) {
       updates.add(createSonarUpdate(release));
     }
     return updates;
   }
 
+  @CheckForNull
+  private Release findPaidReleaseToUpdateTo() {
```

Comment:
This could return an Optional<Release> since there are many cases where null is returned.
It would also make straight forward the check at row 217 with something like 

`findPaidReleaseToUpdateTo().ifPresent(releases::add);`

Reply:
Nice, thanks fixed